src/main/resources/assets/wurst/lang/ru_ru.json

```diff
@@ -39,9 +39,9 @@
   "description.wurst.hack.cavefinder": "Помогает найти пещеры подсвечивая их выбранным цветом.",
   "description.wurst.hack.chattranslator": "Все входящие сообщения в чате будут переводиться гугл переводчиком.",
   "description.wurst.hack.chestesp": "Подсвечивает выбранным цветом сундуки поблизости.",
-  "description.wurst.hack.clickaura": "Автоматически атакует ближайшую сущность когда вы нажимаете ЛКМ.\n\n§c§lВНИМАНИЕ:§r ClickAura'ы обычны выглядят более подозрительно чем Killaura'ы и легче обнаруживаются плагинами. Вместо этого рекомендуется использовать Killaura или TriggerBot.",
+  "description.wurst.hack.clickaura": "Автоматически атакует ближайшую сущность когда вы нажимаете ЛКМ.\n\n§c§lВНИМАНИЕ:§r ClickAur'ы обычны выглядят более подозрительно чем Killaur'ы и легче обнаруживаются плагинами. Вместо этого рекомендуется использовать Killaura или TriggerBot.",
   "description.wurst.hack.clickgui": "Интерфейс с панелями.",
-  "description.wurst.hack.crashchest": "Создает сундук который банит того у кого будет слишком много копий такого сундука в инвентаре. §c§lВНИМАНИЕ:§r §cЭто необратимо. Используйте с осторожностью!§r\n\nЕсли вместо кучу таких копий засунуть в сундук, то любой кто его откроет вылетит из сервера(не навсегда).",
+  "description.wurst.hack.crashchest": "Создает сундук который банит того у кого будет слишком много копий такого сундука в инвентаре. §c§lВНИМАНИЕ:§r §cЭто необратимо. Используйте с осторожностью!§r\n\nЕсли кучу таких копий засунуть в сундук, то любой кто его откроет, вылетит из сервера(не навсегда).",
```

Comment:
punctuation is wrong, should be "—" not comma

Reply:
Thanks, fixed